app/controllers/custom_bigbluebutton_rooms_controller.rb

```diff
@@ -73,7 +73,7 @@ def join_options
     # or if the feature to automatically set the record flag is disabled in the site
```

Comment:
This action `join_options` can be removed from the controller. It renders the view with the "record" checkbox, but that view should not be rendered anymore (and can also be removed).